stable/pgadmin/README.md

```diff
@@ -67,6 +67,7 @@ The command removes nearly all the Kubernetes components associated with the cha
 | `nodeSelector` | Node labels for pod assignment | `{}` |
 | `tolerations` | Node tolerations for pod assignment | `[]` |
 | `affinity` | Node affinity for pod assignment | `{}` |
+| `  PGADMIN_CONFIG_ENHANCED_COOKIE_PROTECTION ` |Allows pgAdmin4 to create session cookies based on IP address | `{False}` |
```

Comment:
Can you please add this as followed under the already existing env:
```
| `env.email` | pgAdmin default email | `chart@example.local` |
| `env.password` | pgAdmin default password | `SuperSecret` |
| `env.enhanced_cookie_protection` | Allows pgAdmin4 to create session cookies based on IP address | `false` |
```

Reply:
done !